pydantic_ai_slim/pydantic_ai/_cli.py

```diff
@@ -151,9 +154,44 @@ def cli(  # noqa: C901
     parser.add_argument('--no-stream', action='store_true', help='Disable streaming from the model')
     parser.add_argument('--version', action='store_true', help='Show version and exit')
 
+    if prog_name == 'clai':
+        parser.add_argument(
+            '--web',
+            action='store_true',
+            help='Launch web chat UI for the agent (requires --agent)',
```

Comment:
I don't think it should require an agent, can't we use `Agent()` and require `-m` for model?